static/css/hugo-easy-gallery.css

```diff
@@ -1,144 +1,20 @@
-/*
-Put this file in /static/css/hugo-easy-gallery.css
-Documentation and licence at https://github.com/liwenyip/hugo-easy-gallery/
```

Comment:
You removed license note?

Reply:
I left 2 lines from original css file out of 150, do you think it's needed? :)